README.md

```diff
@@ -1,13 +1,21 @@
 # stackup2-barefoot-frontend
 
-[![Build Status](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-frontend.svg?branch=develop)](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-frontend)
-
-## To run it locally
-### A.Pre-requisites : 
-* To be able to test this project locally, you need to have the following tools on your local machine:
-> * You need to have [NodeJS version between `6` higher](https://nodejs.org/en/) and [npm version `6` or higher](https://www.npmjs.com/) or [yarn v1.15.0 or higher](https://yarnpkg.com/getting-started/install)
-> * Your local machine needs to be able to run terminal, Powershell, or bash commands
-### B. Steps
+[![Maintainability](https://api.codeclimate.com/v1/badges/a99a88d28ad37a79dbf6/maintainability)](https://codeclimate.com/github/codeclimate/codeclimate/maintainability) [![Build Status](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-frontend.svg?branch=develop)](https://travis-ci.org/Stackup-Rwanda/stackup2-barefoot-frontend)
+
+## General Overview
+*   This Web Application intend to help those who mind to travel by requesting trip and book accommodation in a simplified way.
```

Comment:
`Make company global travel and accommodation easy and convenient for the strong workforce of savvy members of staff, by leveraging the modern web.` 

on this text, I would prefer to use the project's vision as stated above